btor/src/boolector_solver.cpp

```diff
@@ -348,6 +362,20 @@ Term BoolectorSolver::get_value(Term & t) const
   return result;
 }
 
+TermVec BoolectorSolver::get_unsat_core()
+{
+  TermVec core;
+  BoolectorNode ** bcore = boolector_get_failed_assumptions(btor);
+  BoolectorNode ** bcore_iter = bcore;
+  while (*bcore_iter)
+  {
+    core.push_back(std::make_shared<BoolectorTerm>(
+        btor, boolector_copy(btor, *bcore_iter)));
+    bcore_iter++;
```

Comment:
minor: change this to `++bcore_iter;`

Reply:
I actually simplified that whole function even more. It doesn't need a separate iterator, because that array doesn't need to be freed (Boolector manages the memory of the core). 13332aa639fd719a67f459b665a16a240d47a984